relay-statsd/src/lib.rs

```diff
@@ -244,12 +266,21 @@ pub fn init<A: ToSocketAddrs>(
         }
     );
 
-    let statsd_client = if aggregate {
+    let deny_config = DenyTagConfig {
+        starts_with: match config.allow_high_cardinality_tags {
+            true => vec![],
+            false => vec!["hc_".to_owned()],
```

Comment:
Curious, does `.` as a separator not work, that's what we sometimes use for error tags.